plugins/polio/api.py

```diff
@@ -2018,6 +2019,31 @@ def list(self, request):
         return Response({"results": results})
 
 
+@swagger_auto_schema(tags=["lqasimcountries"])
+class CountriesWithLqasIMConfigViewSet(ModelViewSet):
+    http_method_names = ["get"]
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
+    results_key = "results"
+    serializer_class = OrgUnitDropdownSerializer
+    ordering_fields = ["name", "id"]
+    filter_backends = [
+        filters.OrderingFilter,
+        DjangoFilterBackend,
+    ]
+
+    def get_queryset(self):
+        category = self.request.query_params.get("category")
+        configs = Config.objects.filter(slug=f"{category}-config").first().content
+        country_ids = [config["country_id"] for config in configs]
```

Comment:
@quang-le I think here you should make another filter on countries which have datastore(JsonDataStore). Because the request is made on datastore when a country is selected.
![Peek 2023-07-20 11-01](https://github.com/BLSQ/iaso/assets/19631540/27a5714d-4448-4b91-8714-eb836fe31bcb)